src/SFA.DAS.AANHub.Database/Tables/Feedback.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Feedback]
+(
+    [AttendanceId] UNIQUEIDENTIFIER FOREIGN KEY REFERENCES [dbo].[Attendance](Id),
+    [AttributeId] INT NOT NULL, 

```

Comment:
Foreign key?

Reply:
Feedback is against an Attendance record

Reply:
But the AttributeId should match in the Attribute table correct?

Reply:
Yes. Done